docs-kits/kits/PURIS Kit/page_software-development-view.md

```diff
@@ -0,0 +1,932 @@
+BPNL policies. For a detailed description,
+see [Business Partner Validation Extension](https://github.com/eclipse-tractusx/tractusx-edc/tree/main/edc-extensions/bpn-validation).
+
+#### BPNL Access Policy
+
+This policy allows limiting access to a data offer based on a list of specific BPNLs. This translates to the following
+functionality:
+
+- The data offer creator will be able to create a policy listing all the BPN that can access the data offer.
+- This means that only the connectors registered in the Catena-X network with the BPNL listed in the policy can see the
+  data offer and accept it (for the creation of data contracts and subsequent data exchange).
+
+#### Implementation Hint
+
+Examples including a JSON payload for a BPNL group access policy are described in
+the [Business Partner Validation Extension](https://github.com/eclipse-tractusx/tractusx-edc/tree/main/edc-extensions/bpn-validation).
+
+This reference implementation also provides an extension to add the BPNL of the requesting party to the header of a
+proxied request. As a result, during implementation one can use this bpnl to design the internal submodel api design.
+
+### Usage Policies
+
+To decide which company can use the data asset under specific conditions, usage policies (also referred to as contract
+policies) are used. Therefore, they are more specific than access policies and only used just after access is granted.
+Currently, the usage policies aren't technically enforced but based on a legal framework agreements. Signing of
+framework agreements can be enforced during negotiation depending on the connector implementation.
+
+Policies are defined based on the [W3C ODRL format](https://www.w3.org/TR/odrl-model/). This allows a standardized way
+of formulating policy payloads. It further allows to stack different constraints with the `odrl:and` operator.
+Therefore, every data provider can decide on his or her own under which conditions their data assets are shared in the
+network. It is recommended to restrict the data usage for all exchanged data standards. An example of one usage policy
+containing three different constraints is shown and described in the following:
+
+```json
+{
+  "@context": [
+    "http://www.w3.org/ns/odrl.jsonld",
+    {
+      "edc": "https://w3id.org/edc/v0.0.1/ns/",
+      "cx-policy": "https://w3id.org/catenax/policy/"
+    }
+  ],
+  "@type": "PolicyDefinitionRequestDto",
+  "@id": "<POLICY-ID>", // Important for the contract definition
+  "edc:policy": {
+    "@type": "Set",
+    "profile": "cx-policy:profile2405",
+    "permission": [
+      {
+        "action": "use",
+        "constraint": {
+          "@type": "LogicalConstraint",
+          "and": [
+            {
+              "@type": "LogicalConstraint",
+              "leftOperand": "cx-policy:FrameworkAgreement",
+              "operator": "eq",
+              "rightOperand": "Puris:1.0"
```

Comment:
Did someone checked this part? In my understanding the FrameworkAgreement "Puris:1.0" will be deprecated with the release of Jupiter.

see: https://github.com/catenax-eV/cx-odrl-profile/blob/main/profile.md

> Puris:1.0
> Valid from: 2024-06-20
> Valid until: 2024-10-16
> Status: deprecated

Reply:
@dominikoeh,
thank you for including the feedback. Do you have an update on this issue?

Reply:
We're not going to change this. It has been communicated as follows:
- Standards for Data Sovereignty change for R24.08
- Use Case Standards shall be updated for R24.12

The KIT depends on the use case standard linking data sovereingty for R24.08. Thus, no change expected.

Also INT tests have been performed like this for the Software Artifacts.

TBH: I'm not happy to have the data sovereignty update included in this eco system release as it's inconsistent within the release

Reply:
Can be set to resolved from my point of view.